docs/api-testing/sauce-connect.md

```diff
@@ -40,71 +41,16 @@ If your APIs exist behind a firewall on your private network, you'll need to use
   ```bash
   bin/sc -c api-config.yaml
   ```
+5. If the tunnel was launched successfully, you'll see a [CLI response like this](/secure-connections/sauce-connect/quickstart/#verifying-a-tunnel), indicating that you can start your tests.
 
-:::tip
-<details><summary><small>If successful, you'll see a response like this, indicating that you may start your tests.</small></summary>
+## Run an API Test with a Tunnel
 
-```bash title="Success Response"
+6. Now that you're set up to run API tests through a Sauce Connect Proxy tunnel, follow the directions under [Quickstart](/api-testing/quickstart/) to Create a Test (using your local API endpoint), Send a Request, Generate Test.
+7. Tunnels can be selected in all of the places where you can run tests: from the **HTTP Client**, **Schedule** tab, and the **Compose** tab (Composer). Under **Run Configuration**, click the Sauce Connect Proxy tunnels dropdown menu (which defaults to **No Tunnel**), then click the name of your tunnel.<br/><img src={useBaseUrl('img/api-fortress/2021/09/sauceconnect-nav1.png')} alt="API Testing Sauce Connect Nav" width="200"/>
```

Comment:
@kimsauce you can run a test also from a Test List so SC is available also there